src/applications/vaos/components/layouts/CCLayout.unit.spec.jsx

```diff
@@ -244,46 +249,16 @@ describe('VAOS Component: CCLayout', () => {
     it('should display CC layout', async () => {
       // Arrange
       const store = createTestStore(initialState);
-      const appointment = {
```

Comment:
The goal here is to use the MockAppointment object which defines defaults, formatting, etc. for an appointment in one place.